packages/website/src/features/Deploy/PublishUtility.tsx

```diff
@@ -227,16 +227,19 @@ export default function PublishUtility(props: {
               to Ethereum or OP Mainnet to publish this package:
             </Text>
             <UnorderedList mb={4}>
-              {publishers.map((publisher) => (
+              {publishers.map(({ publisher, chainName }) => (
                 <ListItem key={publisher} mb={1}>
                   <Text
                     display="inline"
                     fontFamily="mono"
                     fontWeight={200}
                     color="gray.200"
+                    fontSize="xs"
                     key={`publisher-${publisher}`}
                   >
-                    {`${publisher.substring(0, 8)}...${publisher.slice(-6)}`}
+                    {`${publisher.substring(0, 8)}...${publisher.slice(
+                      -6
+                    )} (${chainName})`}
```

Comment:
Hey @saeta-eth! you could use this util in order to achieve this 
https://github.com/usecannon/cannon/blob/55548f79b063422a7321ec48b54e49ce576bdaae/packages/website/src/helpers/ethereum.ts#L161